openapi/api/v1/openapi.yaml

```diff
@@ -15,6 +15,7 @@ externalDocs:
   url: https://github.com/ecosyste-ms/repos
 servers:
 - url: https://repos.ecosyste.ms/api/v1
+- url: https://data.code.gouv.fr/api/v1
```

Comment:
is this intended to be in main?

Reply:
Not in this PR, but yes we would like to ask you if we could add it, to allow our users to use the API while avoiding patching before deployment. I revert on the PR.